src/Makefile

```diff
@@ -54,7 +54,7 @@ OBJ_FILES = alloc.o bcdf.o bcextr.o bcfree.o bc.o bcrecyc.o bcrelax.o bcshk.o bc
     setup.o solver.o startmpi.o stats.o step.o target_reystress.o tbforce.o updateghost.o utility.o utyibm.o \
     visflx.o writedf.o writefield.o writefieldvtk.o writegridplot3d.o writerst.o \
     writestatbl.o writestatchann.o writestat.o writestatzbl.o write_wallpressure.o visflx_stag.o \
-	write_probe_data.o bcblow.o dissipation.o
+	bcblow.o dissipation.o dealloc.o
```

Comment:
basically just adds `dealloc` and removes `write_probe_data` files from the build.

* `dealloc` deallocates all arrays if they exist. This is so we can start the solver again without needing to restart the python process. Otherwise, the fortran code would attempt to reallocate these arrays. This would be a problem because (1) not enough memory (2) they are already allocated so it would error
* `write_probe_data` is legacy from when I first started working on streams. Since you can now (in theory) run streams from python, exporting data with python IO routines w/ h5py is much much simpler than doing it in fortran